service_storagefacility/src/it/scala/no/uio/musit/microservice/storagefacility/resource/KdReportResourceIntegrationSpec.scala

```diff
@@ -62,14 +62,51 @@ class KdReportResourceIntegrationSpec extends MusitSpecWithServerPerSuite {
 
         val report = wsUrl(KdReportUrl).get.futureValue
 
-        (report.json \ "totalArea").as[Int] mustBe 42
-        (report.json \ "perimeterSecurity").as[Int] mustBe 42
-        (report.json \ "theftProtection").as[Int] mustBe 42
-        (report.json \ "fireProtection").as[Int] mustBe 42
-        (report.json \ "waterDamageAssessment").as[Int] mustBe 0
-        (report.json \ "routinesAndContingencyPlan").as[Int] mustBe 0
+        (report.json \ "totalArea").as[Int] mustBe 41
```

Comment:
I see you're validating the fields here as `Int`, while the datatype `KdReport` defines them as `Double`. Further down on line 92++ you're validating with the correct type.